src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -28,14 +35,31 @@ public SwerveSubsystem(
     if (Utils.isSimulation()) {
       startSimThread();
     }
+
+    Translation2d[] kinematicsTranslations = new Translation2d[4];
+    for (int i = 0; i < 4; i++) {
+      kinematicsTranslations[i] = new Translation2d(modules[i].LocationX, modules[i].LocationY);
+    }
+    kinematics = new SwerveDriveKinematics(kinematicsTranslations);
+
+    AutoBuilder.configureHolonomic(
+        () -> super.getState().Pose,
+        (a) -> super.seedFieldRelative(a),
```

Comment:
We should make sure we're also seeding the gyro with the right values beforehand - seems like CTRE swerve skips that

Looking into it more, seems like this is completely unsupported
We might have to do some kind of manual offset? Like '23? idk